src/tests/e2etest/E2ETest.cs

```diff
@@ -262,6 +262,13 @@ public static void AreEqualByJson(object expected, object actual)
             Assert.AreEqual(expectedJson, actualJson);
         }
 
+        public static bool IsSameByJson(object expected, object actual)

```

Comment:
Unnecessary reference, not seeing any usage.

Reply:
Thanks. Removed.